core/tauri-utils/src/platform.rs

```diff
@@ -239,9 +239,14 @@ pub fn resource_dir(package_info: &PackageInfo, env: &Env) -> crate::Result<Path
   let exe_dir = exe.parent().expect("failed to get exe directory");
   let curr_dir = exe_dir.display().to_string();
 
-  if curr_dir.ends_with(format!("{MAIN_SEPARATOR}target{MAIN_SEPARATOR}debug").as_str())
-    || curr_dir.ends_with(format!("{MAIN_SEPARATOR}target{MAIN_SEPARATOR}release").as_str())
```

Comment:
This check is broken anyways, it doesn't take into account that cargo supports arbitray profile names, cc @lucasfernog

Reply:
yeah, pushed a change. I think checking if .cargo-lock exists should be enough